hms-app/src/pages/YourIdea.tsx

```diff
@@ -14,6 +14,7 @@ function YourIdea() {
   return (
     <>
       <h1>this is the your idea page</h1>
+      <a href="/Your_Idea/Create_New_Idea">Create new idea</a>
```

Comment:
this path seems a bit odd, I would think more like `/idea/create` or something :thinking: